src/hash.py

```diff
@@ -0,0 +1,56 @@
+"""Create a hashtable."""
+
+
+class Hash(object):
+    """Create a hashtable. Its functions are as follows.
+    get(key) - should return the value stored with the given key.
+    set(key, val) - should store the given val using the given key.
+    _hash(key) - should hash the key provided.
+    """
+
+    def __init__(self, size, hash_type='add'):
+        """Instantiate hashtable instance."""
+        self._size = size
+        self._hashtable = [[] for n in range(size)]
+        self._hash_type = hash_type
```

Comment:
Nicely done. Although, consider what you might want to happen if a user tries to use the api with a nonexistant hash.